app/src/main/java/net/aiscope/gdd_app/ui/microscope_quality/MicroscopeQualityActivity.kt

```diff
@@ -29,38 +28,42 @@ class MicroscopeQualityActivity : AppCompatActivity(), MicroscopeQualityView, Ca
     @Inject
     lateinit var presenter: MicroscopeQualityPresenter
 
+    private lateinit var binding: ActivityMicroscopeQualityBinding
+
     override fun onCreate(savedInstanceState: Bundle?) {
         AndroidInjection.inject(this)
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_microscope_quality)
 
-        lifecycleScope.launch {
-            presenter.showScreen()
-        }
+        binding = ActivityMicroscopeQualityBinding.inflate(layoutInflater)
+        with(binding) {
+            setContentView(root)
 
-        setSupportActionBar(toolbar)
-        attachCaptureFlowToolbar(toolbar)
+            setSupportActionBar(toolbar.toolbar)
```

Comment:
I am not quite sure what gets referenced by `toolbar.toolbar` either.

Reply:
Fair enough. The first `toolbar` is the name of the field that points to the `ToolbarBinding` of the included `toolbar.xml` layout, whereas the second `toolbar` is the Toolbar itself in the included layout. The names come from the IDs that were used. 
I renamed the first `toolbar` to `toolbarLayout` to make it clearer, see 28e9b03a080649890b2e5d78efe9107c75a51432

Reply:
💯